src/dvc_render/vega_templates.py

```diff
@@ -157,10 +157,7 @@ class BarHorizontalSortedTemplate(Template):
                 "sort": "-x",
             },
             "yOffset": {"field": "rev"},
-            "color": {
-                "field": "rev",
-                "type": "nominal",
-            },
+            "color": Template.anchor("color"),
```

Comment:
@dberenbaum @daavoo what are the chances that `bar_horizontal_sorted` / `bar_horizontal` will be fed different data from different sources? Any idea how we would/could handle this?

Reply:
Maybe we can use the row or column property from https://vega.github.io/vega-lite/examples/facet_grid_bar.html

Reply:
Would look something like this:

<img width="418" alt="image" src="https://github.com/iterative/dvc-render/assets/37993418/aac627e4-7f6f-4701-8158-d59e07f5ef0d">

Reply:
add `"row":  Template.anchor("row")`